package.json

```diff
@@ -61,6 +61,7 @@
     "react-toggle-dark-mode": "^1.1.1",
     "react-tooltip": "^5.21.5",
     "react-use-query-param-string": "^2.0.10",
+    "sharp": "^0.32.6",
```

Comment:
Pourquoi l'utilisation de sharp ?

Reply:
C'est recommandé par nextjs pour les images => https://nextjs.org/docs/messages/sharp-missing-in-production